### DISCUSSION TITLE
先前同时有Caddy和Nginx的配置没有了吗

### DISCUSSION THREAD

**Discussion by alwzying** [lxhao61/integrated-examples#214] [Q&A] [answered]
先前有Caddy和Nginx共存的方案之后不会有了吗

**Comment by lxhao61** [comment, marked as answer]
是的。[Caddy SNI 性能优于 Nginx SNI 性能](https://github.com/lxhao61/integrated-examples/discussions/186)。之前仅 Nginx 支持 UDP 定向转发，可支持 NaiveProxy 的 HTTP/3 代理应用。当 Caddy 也支持 UDP 定向转发，也可支持 NaiveProxy 的 HTTP/3 代理应用后，就没必要 Caddy 和 Nginx 共存了。
